app/utils/simplified-account/format/format-validation-errors.js

```diff
@@ -1,7 +1,9 @@
+const { kebabCase } = require('change-case')
+
 module.exports = validationResult => {
   const errorSummary = validationResult.array().map(error => ({
     text: error.msg,
-    href: `#${error.path}`
+    href: `#${kebabCase(error.path)}`
```

Comment:
What's the reason for this change?

Reply:
ids in html are kebab case, names are camel case. path here references the name of the input so needs converting